src/components/article/Article.vue

```diff
@@ -8,22 +8,32 @@
             <VsRow>
                 <VsCol
                     cols="12"
-                    class="col-xxl-10 offset-xxl-1"
+                    :class="!businessSupport && 'col-xxl-10 offset-xxl-1'"
```

Comment:
These booleans feel like a slightly confusing way of doing these conditional classes to me. It makes some sense to not use a computed as there are so many of them, but I feel like it might be a bit more obvious what's going on with a ternary (businessSupport ? '' : 'col-xxl-etc)